scrapers/scrape_so.sh

```diff
@@ -11,7 +11,22 @@ d = sc.filter("Situation Kanton Solothurn.*Stand|Anzahl positiv getesteter Erkra
 """
  <p class="bodytext"><strong>Situation Kanton Solothurn (Stand 23.03.2020, 12:00)</strong></p><ul><li>Anzahl positiv getesteter Erkrankungsfälle: 95 Personen</li> 	<li>Verstorben:<strong> </strong>1 Person</li></ul><p class="bodytext"> </p></div></div>
 """
+# 2020-04-02
+"""
+ <p class="bodytext"><strong>Situation Kanton Solothurn (Stand 02.04.2020, 0:00 Uhr)</strong></p><ul><li>Anzahl positiv getesteter Erkrankungsfälle: 227 (+11 im Vergleich zum Vortag)</li> 	<li>Verstorbene Personen:<strong> </strong>3 (keine Veränderung im Vergleich zum Vortag)</li></ul><p class="bodytext"> </p></div></div>
+"""
 
 print("Date and time:", sc.find(r'\(Stand ([^\)]+)\)<', d))
 print("Confirmed cases:", sc.find(r'Anzahl positiv getesteter Erkrankungsfälle: ([0-9]+) ', d))
-print("Deaths:", sc.find('Verstorben:(<strong> <\/strong>)?([0-9]+) ', d, group=2))
+
+deaths_strings = [
+    'Verstorben:(<strong> <\/strong>)?([0-9]+) ',
+    'Verstorbene Personen:(<strong> <\/strong>)?([0-9]+) ',
+]
```

Comment:
Please combine those in one regex, it seems the part "e Personen" is optional, everything else is the same. Then there is no need for a loop.

Reply:
Done